client/src/components/MainPage/TranslationsPage/DonateModal/AudioTranslation/index.jsx

```diff
@@ -18,11 +41,24 @@ class AudioTranslation extends Component {
     this.setState({ selectedFile: files[0] });
   };
 
+  stop = () => {
+    const { recording } = this.state;
+    if (recording) {
+      const { player } = this.refs;
+      this.setState({ recording: false });
+      this.mediaRecorder.stop();
+      const audioBlob = new Blob(this.chunks);
+      const audio = new File(this.chunks, 'record.mp3', { type: 'audio', lastModified: Date.now() });
+      const audioUrl = URL.createObjectURL(audioBlob);
+      player.src = audioUrl;
+      player.classList.remove('no__record');
+      this.setState({ selectedFile: audio });
+    }
+  }
+
   render() {
-    const {
-      error, onChange, generateFormData,
-    } = this.props;
-    const { selectedFile } = this.state;
+    const { error, onChange, generateFormData } = this.props;
+    const { selectedFile, recording } = this.state;
     return (
       <div className="donate__audio">
         <label className="file__label">
```

Comment:
I would suggest the following name.
```suggestion
        <label className="audio__file-label">
```